editor/index.js

```diff
@@ -39,6 +39,9 @@ if ( dateSettings.timezone.string ) {
 window.jQuery( document ).on( 'heartbeat-tick', ( event, response ) => {
 	if ( response[ 'rest-nonce' ] ) {
 		window.wpApiSettings.nonce = response[ 'rest-nonce' ];
+		if ( 'undefined' !== typeof wp.api.endpoints.at( 0 ) ) {
```

Comment:
Without context, I have no idea what this is meant to be doing. Can we adapt some of the "Types of changes" text into a comment here?

Reply:
I added some inline comments here. I also removed the undefined check here to clean up the code and because at this point I think we can assume the wp.api.endpoints.at( 0 )will be set, I can restore if you think a more defensive approach is appropriate.